docs/howto/windows.txt

```diff
@@ -57,8 +57,8 @@ It is best practice to provide a dedicated environment for each Django project
 you create. There are many options to manage environments and packages within
 the Python ecosystem, some of which are recommended in the `Python
 documentation <https://packaging.python.org/guides/tool-recommendations/>`_.
-Python itself comes with `venv`_ for managing environments which we will use
-for this guide.
+Python itself comes with :mod:`venv` for managing environments which we will
+use for this guide.
```

Comment:
```suggestion
Python itself comes with :doc:`python:tutorial/venv` for managing environments
which we will use for this guide.
```

Reply:
I've added this but also overrode the text. Sphinx's default uses the doc's title which is "Virtual Environments and Packages". The override is the original "venv".